vms/platformvm/state/camino.go

```diff
@@ -548,28 +546,20 @@ func (cs *caminoState) Load(s *state) error {
 	cs.lockModeBondDeposit = mode
 
 	baseFee, err := database.GetUInt64(cs.caminoDB, baseFeeKey)
-	if err == database.ErrNotFound {
-		// if baseFee is not in db yet, than its first time when we access it
-		// and it should be equal to config base fee
-		config, err := s.Config()
-		if err != nil {
-			return err
-		}
-		baseFee = config.TxFee
-	} else if err != nil {
+	switch {
+	case err == nil:
+		cs.baseFee = &baseFee
+	case err != database.ErrNotFound:
```

Comment:
what's the deal with these changes?

Reply:
`state` no longer has access to `cfg config.Config`, so instead of on-load writing value from config to state, if there isn't any, we going for more clean approach by allowing value to be nil and treating no-value as not-found error, which is then handled outside of state